Neos.Neos/Classes/Service/XliffService.php

```diff
@@ -204,7 +221,7 @@ function (\XMLReader $file, $offset, $version) use ($packageKey, &$sources, $sou
      *
      * @param array $arrayPointer
      * @param string $key
-     * @param string $value
+     * @param string|array $value
```

Comment:
Did you check if this works with the `NeosCMS.I18n.translate` API in backend modules?

Reply:
Guess what happens 🙄